.cirrus.yml

```diff
@@ -284,7 +285,8 @@ integration_rootless_task:
 
     gce_instance:
         image_name: "$IMAGE_NAME"
-        cpu: 4
+        cpu: 8
+        memory: "24G"
```

Comment:
Do we actually need 24G of ram. Looking at the memory graph it seems we never get above 2G if cirrus can be trusted. Does so much RAM do anything for the performance compared to 16, 8 or 4G? same for the other places